src/client/opamClientConfig.ml

```diff
@@ -210,6 +210,13 @@ let opam_init ?root_dir ?strict ?solver =
 
   (* (i) get root dir *)
   let root = OpamStateConfig.opamroot ?root_dir () in
+  if Sys.win32
+     (* if default, redirection will be handled by opam init, or should have
+        been handled *)
+     && (root_dir <> None || OpamStateConfig.E.root () <> None)
+     && OpamStd.String.contains_char (OpamFilename.Dir.to_string root) ' ' then
+    OpamConsole.error "You opam root directory contains a space, this may lead \
+                       to several malfunction... bzzz.... nooo💥";
```

Comment:
This is a opening for proposal :)

Reply:
that looks fine by me

Reply:
I like it - although the 💥 should be guarded with a UTF-8 trick (like the camel!)

Reply:
done